src/components/VirtualizedList/__stories__/VirtualizedList.stories.mdx

```diff
@@ -4,7 +4,8 @@ import { generateItems } from "./virtualizedList.components";
 import { useCallback } from "react";
 import { Flex } from "../../";
 import styles from "./virtualizedList.module.scss";
-import { Tip, UsageGuidelines } from "../../../storybook";
+import { Tip } from "../../../storybook";
```

Comment:
what is Tip? why is it here and not in vibe-storybook-components?

Reply:
Tip is implemented via `AttentionBox` component. We can duplicate it in `vibe-storybook-components`, but it requires some efforts, so I've decided not to put it into the first version

Reply:
ok, but let's put it as a TODO somewhere, so it can be a good first task.

Reply:
Added it to our backlog